kalite/coachreports/features/steps/coachreports.py

```diff
@@ -101,15 +100,15 @@ def create_some_learner_data():
         ("attempted", 50, 10),
         ("struggling", 30, 25),
     )
-    exercises = random.sample(get_exercise_cache().keys(), len(attempt_states))  # Important they are *distinct*
```

Comment:
More efficient to randomly sample at the database level, so this convenience function handles that.